### PR TITLE
Upgrade to Grafana 1.9.1

### PR DESCRIPTION
**Pull request by ashb**

### CODE REVIEW

**Comment by ashb** [comment]
This PR will need rebasing once #49 has been merged — this PR also changes the config file so basing off that makes the rebase straight forward rather than needing to resolve a conflict.)

File: metrics/grafana.sls [hunk 1]
```diff
@@ -6,29 +5,37 @@ include:
...
+/srv/grafana/.ssh:
+  file.absent
+
+/srv/grafana:
+  file.directory:
+    - recuse: True
+
+grafana-download:
+  archive.extracted:
+    - name: /srv/graphite
```
Comment (mattmb) [inline metrics/grafana.sls]: grafana?

Comment (ashb) [inline metrics/grafana.sls]: WHOOPS!. Good spot :)

Comment (ashb) [inline metrics/grafana.sls]: @mattmb fixed, rebased and pushed that one.

**Comment by mattmb** [comment]
LGTM, have you tested it using monitoring-deploy?

**Comment by ashb** [comment]
I haven't on that, I will do so now. (I have tested on PQ's box but should test it works there too)

**Comment by ashb** [comment]
@mattmb yes, just tested. Grafana is working fine there.

**Pull request merged**